src/main/java/com/hubspot/jinjava/lib/filter/BetweenTimesFilter.java

```diff
@@ -73,15 +77,26 @@ private ZonedDateTime getZonedDateTime(Object var, String position) {
     } else {
       JinjavaInterpreter interpreter = JinjavaInterpreter.getCurrent();
 
-      interpreter.addError(
-        TemplateError.fromMissingFilterArgException(
-          new InvalidArgumentException(
-            interpreter,
-            getName() + " filter called with null " + position,
-            getName()
+      if (var == null) {
+        interpreter.addError(
+          TemplateError.fromMissingFilterArgException(
+            new InvalidArgumentException(
+              interpreter,
+              getName() + " filter called with null " + position,
+              getName()
+            )
           )
-        )
-      );
+        );
+
+        FeatureActivationStrategy feat = interpreter
+          .getConfig()
+          .getFeatures()
+          .getActivationStrategy(FIXED_DATE_TIME_FILTER_NULL_ARG);
+
+        if (feat.isActive()) {
+          var = ((DateTimeFeatureActivationStrategy) feat).getActivateAt();
```

Comment:
These are not safe casts

Reply:
In general, true, but for this particular feature it will be.

Reply:
If the feature is changed to `FeatureStrategies.ACTIVE`, this will blow up at runtime

Reply:
Yes. But for compatibility reasons, this has to return a valid date. So it's either a casting exception or some other exception.